communications.md

```diff
@@ -2,95 +2,62 @@
 
 ## General guidelines
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+Communication is critical to any organization and in doing so, we should be considerate of other people's production processes and time.
+We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite, Asana and Emails.
 
-## First line of Communication: Texting
+Our communication tools encompasses both formal and informal communications as such we may want to understand:
 
-We use a few tools to help team members communicate with one another: Slack, G Suite and Asana. Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, use Slack, Gmail or Asana to contact another team member asynchronously (in an intermittent manner).
+- your audience 
+- the size of the audience 
+- content of message and reusability (if the content of the message will come in handy in the future) 
 
-Budget communication time. Try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
+and then choose the appropriate communication tools to ensure effective communication.
 
-#### - Internal & External Messaging Tool: Slack -
-Opt-in: Join channels that you are interested in or obligated to.
+Time management is key to success and so is effective communication. Sometimes communication tools can reduce productivity time hence, we can individually implement suitable patterns to ensure we effciently and effectively manage our time.
 
-Setting up Notifications: Once you're part of a channel, set up your notifications to suit your needs:
```

Comment:
this and below should not be deleted. but rephrased so that this can be a suggested way of utilizing the tools to avoid drowning into the tool and making the tools counter-productive.

Reply:
I feel like slack notification management can be a small sub markdown page all on it's own (snooze, alerts, silent etc.) and linked here.